distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,230 @@
 
-// https://codice.atlassian.net/browse/DDF-2651
-placeholder text
+Introduction: The ${catalog-ui} represents the most advanced search interface available with ${branding}.
+It provides a metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
+
+[NOTE]
+====
+For more detail on any feature or button within the Catalog UI, click the `?` button in the upper right of the screen; then, hover over any item on the screen and a contextual tooltip will to define its purpose.
+To exit this mode, click the `?` again or press `escape`.
+====
+
+=== Accessing ${catalog-ui}
+
+The default location for the ${catalog-ui} is ${secure_url}/search/catalog
+
+=== Catalog UI Guest Users
+
+Guest users not signed in to ${branding} will have access to search functions, but all workspace configuration and settings will only exist locally and not be available for sharing.
+
+=== Workspaces in ${catalog-ui}
+
+Within the ${catalog-ui}, workspaces are collections of settings and standing searches that can be shared between users and stored to be accessed repeatedly.
+
+==== Creating a Workspace in ${catalog-ui}
+
+Before searching in ${branding}, at least one workspace must be created.
+Use an existing template or create a blank workspace.
+
+.Default Workspace Templates
+Local:: an example of a local search.
+Federated:: an example of a federated search.
+Location:: an example of a geographically constrained search.
+Temporal:: an example of a temporally constrained search.
+
+==== Configuring a Workspace in ${catalog-ui}
+
+Configure each workspace with standing queries and share options.
+
+.Adding searches
+. Select the default name of the workspace (*New Workspace* or template name) to change to a unique name.
+. If *Blank* workspace was selected, select *Add Search* to add the first search. Each workspace can have up to ten searches.
+.. Select *Basic* Search to select simple search criteria, such as *text*, *time*, and *location*.
+.. Select *Advanced* Search to access a query-builder for more complex queries.
+
+.Navigator Menu Options
+* *Workspaces*: View all available workspaces.
+* *Upload*: Interface to add resources to catalog.
+* *Open Workspaces* (if applicable): Shows a list of open workspaces.
+
+.Workspace Menu Options
+* Select the *Menu* button next to the Workspace name.
+** *Save*: Save changes to the workspace.
+** *Run All Searches*: Start all saved searches within this workspace.
+** *Cancel All Searches*: Cancel all running searches.
+** *Open in New Tab*: Opens this workspace in a separate tab.
+** *View Sharing*: View and edit settings for sharing this workspace. (User must be signed in to share or view shared workspaces.)
+** *View Details*: View the current details for a cloud-based workspace (User must be signed in).
+** *Duplicate*: Create a copy of this workspace.
+** *Subscribe*: Receive updates on this workspace.
```

Comment:
This should be "Receive email notifications for search results on this workspace (If email notifications have been enabled)."